tests/assertions.js

```diff
@@ -57,3 +54,26 @@ export function shouldBeIssuedVc({issuedVc}) {
       'Expected issuer object to have property id');
   }
 }
+
+function _shouldBeCredentialSubject({credentialSubject}) {
+  // credentialSubject should not be null or undefined
+  should.exist(credentialSubject, 'Expected credentialSubject to exist.');
+  // a credentialSubject can be an Array of objects
+  if(Array.isArray(credentialSubject)) {
+    credentialSubject.length.should.be.gt(
+      0,
+      'Expected credentialSubject to make a claim on at least one subject.'
+    );
+    for(const subject of credentialSubject) {
+      subject.should.be.an(
+        'object',
+        'Expected credentialSubject to be an object.'
+      );
+    }
+    return;
+  }
+  credentialSubject.should.be.an(
+    'object',
+    'Expected credentialSubject to be an object.'
+  );
```

Comment:
It needs to not only be an object, but also have at least one internal property.

You should create a function and share it between this code and the code above that checks to see if it's a valid object AND that it has at least one property defined in the object.

Reply:
Addressed here: https://github.com/w3c-ccg/vc-api-issuer-test-suite/pull/28/commits/f0878d3f9ea47b76d27f093208d13de0ff216d37